03_Modules/Configuration/src/module/module.ts

```diff
@@ -106,6 +107,7 @@ export class ConfigurationModule extends AbstractModule {
   protected _bootRepository: IBootRepository;
   protected _deviceModelRepository: IDeviceModelRepository;
   protected _messageInfoRepository: IMessageInfoRepository;
+  protected bootService: BootNotificationService;
```

Comment:
very minor, should this be `_bootService` to match with all the other repositories and service names? 🤔

Reply:
Sure thing!